rsconnect/main.py

```diff
@@ -276,27 +280,72 @@ def _test_server_and_api(server, api_key, insecure, ca_cert):
     type=click.File(),
     help="The path to trusted TLS CA certificates.",
 )
+@click.option(
+    "--account",
+    "-a",
+    envvar="SHINYAPPS_ACCOUNT",
+    help="The shinyapps.io account name.",
+)
+@click.option(
+    "--token",
+    "-T",
+    envvar="SHINYAPPS_TOKEN",
+    help="The shinyapps.io token.",
+)
+@click.option(
+    "--secret",
+    "-S",
+    envvar="SHINYAPPS_SECRET",
+    help="The shinyapps.io token secret.",
+)
```

Comment:
these new options could be all added under @server_args https://github.com/rstudio/rsconnect-python/blob/72c58f47675c7db84632a0688c49400ca2ea73dc/rsconnect/main.py#L100-L133

Reply:
The only method that will work with shinyapps.io right now is `deploy_manifest`, so I think it would be misleading for rsconnect-python to explicitly accept those options for the other methods.